extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/dto/VaultDbCreds.java

```diff
@@ -0,0 +1,5 @@
+package io.quarkus.vault.runtime.client.dto;
+
+public class VaultDbCreds extends AbstractVaultDTO<VaultDbCredsData, Object> {
```

Comment:
Is the "Creds" part a Vault name? If not, I would prefer we use the full name.

Same remark for all the `Creds`.

Reply:
it is: https://www.vaultproject.io/docs/secrets/databases/index.html#usage

but it does not hurt to settle on credentials instead.
=> all `creds` have been replaced with `credentials`.